modules/platform/nuxeo-platform-video/nuxeo-platform-video-core/src/test/java/org/nuxeo/ecm/platform/video/extension/TestVideoImporterAndListeners.java

```diff
@@ -452,9 +449,46 @@ public void testConfiguration() throws Exception {
         // check storyboard
         @SuppressWarnings("unchecked")
         List<Map<String, Serializable>> storyboard = (List<Map<String, Serializable>>) docModel.getPropertyValue(
-                "vid:storyboard");
+                VideoConstants.STORYBOARD_PROPERTY);
         assertNotNull(storyboard);
         assertEquals(2, storyboard.size());
     }
 
+    @Test
+    @Deploy("org.nuxeo.ecm.platform.video.core:test-video-conversions-enabled.xml")
+    @Deploy("org.nuxeo.ecm.platform.video.core:test-video-listeners-contrib.xml")
+    @Deploy("org.nuxeo.ecm.platform.video.core:video-configuration-override.xml")
```

Comment:
the last `@Deploy("org.nuxeo.ecm.platform.video.core:video-configuration-override.xml")` is needed to have the storyboard (because storyboard check on the duration [here](https://github.com/nuxeo/nuxeo/blob/master/modules/platform/nuxeo-platform-video/nuxeo-platform-video-core/src/main/java/org/nuxeo/ecm/platform/video/VideoHelper.java#L122) and I don't want to test with a big video, the contribution:

`<storyboardMinDuration>0</storyboardMinDuration>` which suits my need 

for the the two I should recheck (i did mainly because the current test on conversion adds them, but i am going to recheck)